sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,132 @@
+from sympy.physics.mechanics import KanesMethod
+
+__all__ = ['JointsMethod']
+
+
+class JointsMethod(object):
+    """
+    Joints method object.
+
+    This object is used to form the equations of motion of a system created
+    using Joints.
+
+    Parameters
+    ----------
+    joints: List of Joints
+        List of all the joints created must be passed as argument.
+    root_body: Body
+        Root body w.r.t which equations of motion much be generated.
+        Is necessary to remove the  ambiguity since user may not
+        connect all bodies using joints and thus, there is not
+        way to find the root_body.
+
+    Examples
+    --------
+    >>> from sympy import Symbol
+    >>> from sympy.physics.mechanics import (Body, PinJoint, SphericalJoint, \
+                                             JointsMethod)
+    >>> root_body = Body('root_body')
```

Comment:
If we decided to create a `Ground` body like you orginally had planned could we avoid having to explicitly pass in the root_body to JointsMethod?

Also, why do you have to pass in the root_body explicitly? Wouldn't it be possible to figure out which body is at the "bottom" of the tree?

Reply:
Thinking about this, I don't think we can know what body should be the ground without the user specifying that.

Reply:
> Thinking about this, I don't think we can know what body should be the ground without the user specifying that.

But what is the use of ground body? In the code I could only see the use of ground body to take that body's frame.

So can't we make user directly give frame?

Reply:
There has to be a single Newtonian reference frame in which you want to calculate the equations of motion. The idea of having a ground body is that you can pick any body to be the netownian frame, calling that "ground".

Reply:
Oh, this seems reasonable! Thanks @moorepants